tests/lct_test.py

```diff
@@ -174,34 +150,6 @@ def _case(case_number, data_dir):
     return i
 
 
-def _vals():
-    dus = []
-    all_fvals = []
-    all_uvals = []
-    for i, inputs in enumerate(((3.0, 69), (5.0, 50), (8.0, 100), (8.0, 100))):
-        du, uvals, fvals = d_f_u_vals(*inputs, i)
-        dus.append(du)
-        all_fvals.append([f for f in fvals])
-        all_uvals.append([u for u in uvals])
-    return dus, all_fvals, all_uvals
-
-
-def _ndiff_files(expect_path, actual_path, diff_file, data_dir):
-    pksubprocess.check_call_with_signals(
-        [
-            "ndiff",
-            actual_path,
-            expect_path,
-            data_dir.join("ndiff_conf.txt"),
-        ],
-        output=str(diff_file),
-    )
-
-    d = pkio.read_text(diff_file)
-    if re.search("diffs have been detected", d):
-        raise AssertionError(f"{d}")
-
-
 def _cast_complex_for_write(number):
```

Comment:
The word `cast` is loaded with meaning that isn't quite what's going on here. Indeed, I think what you would want for "write" is to convert it to a string so _complex_as_str would be a better name, I think, and to change what it produces.